runtime/src/status_cache.rs

```diff
@@ -230,7 +230,21 @@ impl<T: Serialize + Clone> StatusCache<T> {
                 (
```

Comment:
... so, this out-of-diff `pub fn slot_deltas(...)` signature can be `pub(crate) fn slot_delta(...)` or just `fn slot_deltas(...)`?